book/src/storage/transfer-to-object.md

```diff
@@ -1,4 +1,132 @@
-# Transfer to Object?
+# Transfer to Object
 
-The `transfer::transfer` call takes the receiver `address` as the second argument, and while in most
-of the cases it is an account address, it can also be an address of an object. In this case,
+Previously, we have explained how [transfer](./storage-functions.md#transfer) works in relation to
+accounts. However, there is an additional behaviour allowed by the
+[Sui Object Model](./../object/object-model.md) - transferring objects to other objects. If certain
+conditions are met (which we explain in this section), objects can be sent to and _received_ from
+other objects.
+
+<div class="warning">
+
+Warning: recipient object must be implemented in a way that allows receiving. Attempt to send an
+object to an object that does not implement receiving can result in asset loss.
+
+</div>
+
+## Background
+
+Every object in Sui has its own `UID` which is represented by the
+[address type](./../move-basics/address.md). This address can be used to query the object's data,
```

Comment:
This may be a bit confusing to folks, because depending on context, an object's ID can be represented by a `UID`, `ID`, or `address` type. On this page the `address` representation is the most interesting/novel one, but in other circumstances (e.g. for dynamic fields) `UID` is more interesting, and in other circumstances (e.g. ID pointer) `ID` is more interesting.

Here seems like the wrong place to discuss this concept in detail, but it seems like something we should document somewhere.